src/fetch.ts

```diff
@@ -2,5 +2,5 @@ import nodeFetch from 'node-fetch';
 
 export = (
     // istanbul ignore next
-    typeof window === 'undefined' ? (nodeFetch as typeof fetch) : fetch
+    typeof window.fetch === 'undefined' ? (nodeFetch as typeof fetch) : fetch
```

Comment:
As I understand it, this all needs to work in Node.js as well, so you'll still need to check that `typeof window` is not `undefined` before access properties.

Reply:
Yes